app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java

```diff
@@ -22,14 +21,14 @@
  * 
  */
 public class RestClient extends OAuthBaseClient {
-	public static final Class<? extends Api> REST_API_CLASS = FlickrApi.class; // Change this
+	public static final Class<? extends BaseApi> REST_API_CLASS = FlickrApi.class; // Change this
 	public static final String REST_URL = "https://api.flickr.com/services"; // Change this, base API URL
 	public static final String REST_CONSUMER_KEY = "SOME_KEY";       // Change this
 	public static final String REST_CONSUMER_SECRET = "SOME_SECRET"; // Change this
 	public static final String REST_CALLBACK_URL = "oauth://cprest"; // Change this (here and in manifest)
 
 	public RestClient(Context context) {
-		super(context, REST_API_CLASS, REST_URL, REST_CONSUMER_KEY, REST_CONSUMER_SECRET, REST_CALLBACK_URL);
+		super(context, REST_API_CLASS, FlickrApi.instance(), REST_URL, REST_CONSUMER_KEY, REST_CONSUMER_SECRET, REST_CALLBACK_URL);
```

Comment:
This requires the API class to be specified twice now right and changed the constructor, I wonder if there's anyway to avoid specifying "FlickrClient" twice?